plugins/py3embed/py3embed.cpp

```diff
@@ -1197,7 +1312,14 @@ class PythonNamedTupleBuilder : public CInterfaceOf<IFieldProcessor>
     PyObject *getTuple(const RtlTypeInfo *type)
     {
         OwnedPyObject mynamedtupletype = sharedCtx ? sharedCtx->getNamedTupleType(type) : globalState.getNamedTupleType(type);
+        Py_ssize_t len = PyList_Size(args);
```

Comment:
does not seem to be used

Reply:
No, that was part of my first failed attempt to fix.I will remove.